project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,161 @@
+# Waypoint Project Template Example
+
+This is an example of how to create and use project templating in HCP Waypoint.
+
+Pre-Requisites
+
+AWS Account with Admin Permissions
+
+Terraform Enterprise Account with No Code Module Enabled.
+
+## Initial Setup
+
+### Terraform Module Creation
+
+TODO(Teresa): Make this repo public.
+
+1. Fork the `hashicorp/terraform-aws-example-microservice-ecs-all-envs` repository in Github [here](https://github.com/hashicorp/terraform-aws-example-microservice-ecs-allenvs)
+
```

Comment:
It looks like the example no-code module references the TFC org by name: https://github.com/hashicorp/terraform-aws-example-microservice-ecs-allenvs/blob/main/remote_state.tf#L4

The user will need to update that to use their own org id.